openpype/modules/ftrack/tray/ftrack_tray.py

```diff
@@ -48,6 +51,31 @@ def show_login_widget(self):
         self.widget_login.activateWindow()
         self.widget_login.raise_()
 
+    def show_ftrack_browser(self):
+        cur_os = platform.system().lower()
+        settings = get_system_settings()["modules"]["ftrack"]
+        browser_paths = settings["ftrack_browser_path"][cur_os]
+        browser_args = settings["ftrack_browser_arguments"][cur_os]
+        browser_args.append(self.module.ftrack_url)
+        path = ""
+        for p in browser_paths:
+            if os.path.exists(p):
+                path = p
+                log.debug(f"Found valid executable at path: {p}")
+                break
+            else:
+                log.warning(f"Path: {p} is not valid, please \
+                    doublecheck your settings!")
+        if path == "":
+            log.warning("Found no valid executables to launch \
+                Ftrack with. Feature will not work as expected!")
+            return
+        args = " ".join(str(item) for item in browser_args).replace("= ", "=")
+        log.debug(f"Computed arguments: {args}")
+        cmd = f"{path} {args}"
+        log.debug(f"Opening Ftrack Browser...")
+        subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
That was done to open the browser window in -app mode (at least chrome).

Reply:
Also, stupid chrome needs no space between -app= and url, so it would need logic to differentiate chrome from the others. That's why i put that in the settings, where you could specify browser and arguments and control the logic. I understand this is a very cosmetic only feature, it's mainly used to have the window with the ftrack icon in the taskbar so the artist does not close it by mistake when quitting the browser, but we can avoid it if the solution is too overkill for the task i suppose

Reply:
> That was done to open the browser window in -app mode (at least chrome).

Now, I'm not sure if I fully understand the feature. Isn't the goal just to "open url in browser"?

If yes then the Qt approach does that. Qt will ask OS to open the url and OS will care about the rest. You don't have to care about any executables or launching subprocess on your own.
Just replace `show_ftrack_browser` with this and you'll see.
```
def show_ftrack_browser(self):
    from Qt import QtGui

    QtGui.QDesktopServices.openUrl(self.module.ftrack_url)
```

Reply:
![ftrack](https://user-images.githubusercontent.com/19264472/173571364-b9e4d3d4-d4b8-4e66-8ab3-ae04f8c99897.JPG)
it's kinda like this visually. No chrome bars, no url bar, and ftrack icon in the taskbar

Reply:
I've also tried QT but it did not allow inserting arguments, so i've opted with a longer approach with webbrowser.

Reply:
I've just tested this again, it's broken. webbrowser is opening in a blocking way, i can't access the tray after opening.
Let me try a few things

Reply:
Just needed to return the function! Seems to be working. I would need to test it in Linux and chrome or chromium to see if it's working, windows side is peachy